### PR TITLE
removed not used timeline translations

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

**Comment by quisquous** [comment]
Are these really all of them? Are there none in ShB?

**Comment by Akurosia** [comment]
maybe, just maybe, i got distracted with another script that made the mixup in the last PR where i forgot to continue on the removal of not used timelinereplacements.
Give me like 15min and than i will report the missing once

**Comment by Akurosia** [comment]
FYI
i need to investigate on following files: eureka_hydatos, suzaku-ex, tsukuyomi-ex, the_copied_factory, twinning and e1s
They currently cant be parsed due to an issue (suzaku will be a colon issue for `:Tenzen` and the rest most likly a self made issue (e.g. accidently removing a comma)

**Comment by Akurosia** [comment]
Ok except suzaku, everthing could be considired done.
For suzaku, the entry `:Tenzen` breaks the check_translation script. This could be avoided by using either `\\:Tenzen` or just `Tenzen`

File: ui/raidboss/data/03-hw/dungeon/sohm_al_hard.js [hunk 1]
```diff
@@ -188,32 +188,20 @@
       'locale': 'de',
       'replaceSync': {
         'The Wound': 'Wunde',
-        'The Fever': 'Das Fieber',
         'The Lava Tube': 'Lavagrotte',
-        'The Scorpion\'s Tail': 'Schwanzskorpion',
         'The Leightonward': 'Hortigolem',
-        '(?<! )Spore Sac': 'Sporensack',
-        'Small Spore Sac': 'klein[a] Sporensack',
+        'Small Spore Sac': 'klein(?:e|er|es|en) Sporensack',
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/dungeon/sohm_al_hard.js:193]: "remove not used timeline translations" ... but also make some modifications too!

File: ui/raidboss/data/03-hw/raid/a5s.js [hunk 2]
```diff
@@ -289,6 +289,7 @@ let bombLocation = (matches) => {
         '(?<!Hummel)Faust': 'Faust',
         '(?<!Smart)Bomb': 'Bombe',
         'Hummelfaust': 'Hummelfaust',
+        'Gobbledygroper': 'Gobgreifer',
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a5s.js:292]: Sorry, it is hard enough to review removals from the timeline section on its own, but it becomes much harder when you are also making additions and modifications.

If you'd like to clean up unused entries, can you make this patch *just* removals?

Comment (Akurosia) [reply ui/raidboss/data/03-hw/raid/a5s.js]: reverted all additions (except the needed missing translation flag for one of the files)
Once this PR is done, the addiotions will be made again

**Comment by quisquous** [comment]
This is why this is hard to review.  These are the errors you introduced:
```diff
0a1,2
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o6s.js:27 O6S Storms Grip: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the storm's grip\.' (but have regexDe)
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\trial\shinryu-ex.js:407 ShinryuEx Heart: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the worm's heart\.' (but have regexDe)
15a18,19
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o10n.txt:15 text "Out"
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o10n.txt:18 text "Out"
195c199
< C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o6s.js:27 O6S Storms Grip: incorrect timelineReplace for regex, got '^.{14} 03:[0-9a-f]{8}:added new combatant emprise de la temp\xeate\.', needed regexFr '^.{14} 03:[0-9a-f]{8}:added new combatant zone de temp\xeate\.'
---
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o6s.js:27 O6S Storms Grip: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the storm's grip\.' (but have regexFr)
242a247
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\trial\shinryu-ex.js:407 ShinryuEx Heart: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the worm's heart\.' (but have regexFr)
962a968
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o6s.js:27 O6S Storms Grip: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the storm's grip\.' (but have regexJa)
1157a1164
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\trial\shinryu-ex.js:407 ShinryuEx Heart: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the worm's heart\.' (but have regexJa)
1461c1468
< C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o6s.js:27 O6S Storms Grip: incorrect timelineReplace for regex, got '^.{14} 03:[0-9a-f]{8}:added new combatant \u66b4\u98ce\u57df\.', needed regexCn '^.{14} 03:[0-9a-f]{8}:added new combatant \u66b4\u98ce\u9886\u57df\.'
---
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o6s.js:27 O6S Storms Grip: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the storm's grip\.' (but have regexCn)
1466a1474
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\trial\shinryu-ex.js:407 ShinryuEx Heart: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the worm's heart\.' (but have regexCn)
1820a1829
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\raid\o6s.js:27 O6S Storms Grip: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the storm's grip\.' (but have regexKo)
1901a1911
> C:\Users\enne\cactbot\ui\raidboss\data\04-sb\trial\shinryu-ex.js:407 ShinryuEx Heart: missing timelineReplace for regex '^.{14} 03:[0-9a-f]{8}:added new combatant the worm's heart\.' (but have regexKo)
```

**Comment by Akurosia** [comment]
Could you also pls check what happens when you remove Storm's Grip from e6s and run `python .\util\check_translation.py -t de -f o6n.js -gm all` ?
I think this is an issue in the script (thats why it was removed, as my script is a modification of check_translation)

**Comment by quisquous** [comment]
> Could you also pls check what happens when you remove Storm's Grip from e6s and run `python .\util\check_translation.py -t de -f o6s.js -gm all` ?
> I think this is an issue in the script (thats why it was removed, as my script is a modification of check_translation)

```
$ python util/check_translation.py -t de -f o6s.js -gm all
18 "--targetable--" #MISSINGTEXT
641 "Enrage" #MISSINGTEXT
```

Is it possible that your script isn't handling apostrophes correctly?

**Pull request merged**